internal/command/metadata_functions.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 var (
-	ignoredFunctions = []string{"map", "list"}
+	ignoredFunctions = []string{"map", "list", "core::map", "core::list"}
```

Comment:
Just noting here because this change reminded me of this command: do we intend in a later PR to expand `metadata functions` to include the functions offered by installed providers?

Reply:
`metadata functions` doesn't require an initialized working directory, so unfortunately provider functions are not going to be part of that output. Since that is mainly for the language server integration, it's yet to be determined if a separate command to list providers functions is needed (they are fetching schemas anyways if providers are available, which include functions), or maybe having inconsistent output from `metadata` is actually OK.